_posts/payalpatel/2016-01-27-PayalsCh4Exercises.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+author: payalpn
+title: "Payal's Chapter 4 Exercises"
+---
+
+**Chapter 4, Exercise 6:**
+
+<iframe src="https://trinket.io/embed/python/152e6a3732" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Chapter 4, Exercise 7:**
+
+<iframe src="https://trinket.io/embed/python/0f16febc14" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Reflection:**
+While working on these exercises I found that I didn’t have to write a lot of new code.  Since these functions were based off of previous exercises I was able to do some slight adjustments and additions to create the functions.  I would say the most challenging part is making sure blocks are lined up correctly—I tend to get errors when things are nested inside one another if even one line is not indented where it should be.  Overall, I found these exercises interesting, but not to challenging.  
```

Comment:
Ah good call!  I'll talk about indentation hacks in class.

not challenging is OK for the chapter exercises.  Turtles are where it's at :)